nmqtt.nim

```diff
@@ -1045,7 +1047,7 @@ proc connectBroker(ctx: MqttCtx) {.async.} =
     ctx.s = await asyncnet.dial(ctx.host, ctx.port)
     if ctx.sslOn:
       when defined(ssl):
-        ctx.ssl = newContext(protSSLv23, CVerifyNone)
+        ctx.ssl = newContext(protSSLv23, CVerifyNone, ctx.sslCertFile, ctx.sslKeyFile)
```

Comment:
We dont have any test for this yet, but I guess that not specifying any key- or cert-file just sends an empty string, which would have no impact on `newContext`. Can you confirm?

Reply:
Looks like it reacts the same way if it's there or not